.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+fail_fast: true
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-added-large-files
+        args: [--maxkb=10000, --enforce-all]
+      - id: check-executables-have-shebangs
+      - id: check-shebang-scripts-are-executable
+      - id: mixed-line-ending
+      - id: trailing-whitespace
```

Comment:
This line is a duplicate of line 6. If you do a new PR for this, as I requested elsewhere, I would recommend replacing it with `end-of-file-fixer`.